docs/UserGuide.md

```diff
@@ -201,6 +202,45 @@ To check total modular credits:
 2. The result display box will display the message "Total MC has been calculated" along with your total modular credits amount.
 ![CheckMc2](images/CheckMc2.png)
 
+### Checking required modules: `required`
+
+The `required` command allows you to check the modules of the Computer Science curriculum you have yet to take, based on your current GradPad module list.
```

Comment:
Maybe we can use the term `Completed Modules` list since we're using it throughout the UG?

Reply:
Yeah we not calling it current GradPad modules anymore

Reply:
Woops forgot about that part hahaha, thanks for the find, replaced!